recipes-core/everest-chargebyte/everest-chargebyte_git.bb

```diff
@@ -3,7 +3,7 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
 
 SRC_URI = "git://github.com/chargebyte/everest-chargebyte.git;branch=main;protocol=https"
 
-SRCREV = "9709c13993eca6716c9664b05d4de3033a13771f"
+SRCREV = "219caa623f060abfa989bc51b360f60ca31b1800"
 PV = "0.14.0+git${SRCPV}"
```

Comment:
It's not tagged yet, it's just the latest version post-0.13.0.

We do point to moving targets here, don't we? And then tag them on release.

Reply:
I believe the moment we change the PV / project() VERSION (https://github.com/chargebyte/everest-chargebyte/commit/0af174d2ee62c0178f131b2485037aa1939f394f) , the module must be tagged, so that everything aligns

Reply:
Perhaps I got confused in the process.

I have tagged 0.14.0, and also 0.15.0 with the recent changes, especially regarding the interface compatibility.